packages/apfelxx/variables.sh

```diff
@@ -1,5 +1,5 @@
 BRANCH="resummation-scales"
-VERSION="7572c1b"
-URL="https://github.com/vbertone/apfelxx/tarball/$BRANCH"
+VERSION="00238cd"
+URL="https://github.com/vbertone/apfelxx/tarball/$VERSION"
 TARBALL="apfelxx-$BRANCH.tar.gz"
```

Comment:
To make a successful installation I believe you also need:
```suggestion
TARBALL="apfelxx-$VERSION.tar.gz"
```